README.md

```diff
@@ -180,6 +180,12 @@ myNextApplication:
               maxTTL: 10
               defaultTTL: 10
       priceClass: "PriceClass_100"
+      # You can add custom error responses
+      errorPages:
+        - code: 503
+          path: "/503.html"
+          ttl: 5 # optional, minimum ttl the error is cached (default 10)
```

Comment:
minor: perhaps call this `minTTL` to be clearer and also consistent with the other CloudFront inputs (defaultTTL, minTTL, maxTTL`)

Reply:
No problem, fixed :)